themes/jaeger-docs/layouts/partials/navbar.html

```diff
@@ -72,18 +73,20 @@
                 {{ $docPath = .Path -}}
               {{ end -}}
               {{ $docVersion := index (split $docPath "/") 1 -}}
-              <!-- When generating Docs menu for home page, use the latest version, -->
-              <!-- when generating menu for pages inside docs/, use same version as the doc -->
```

Comment:
These two HTML comments turned into blank lines. They were the main source of the remaining (3K) blank lines.